mars/oscar/core.pyx

```diff
@@ -30,14 +30,14 @@ CALL_METHOD_BATCH = 1
 logger = logging.getLogger(__name__)
 
 cdef:
-    bint _log_unhandled_errors = False
-    bint _log_cycle_send = False
+    bint _log_unhandled_errors = True
+    bint _log_cycle_send = True
```

Comment:
Make these to False to avoid expensive error detection.

Reply:
reverted